server/src/pages/shot/view.js

```diff
@@ -540,7 +540,7 @@ class Body extends React.Component {
     } else {
       sendEvent("unset-favorite", "navbar");
       const TWO_WEEKS_IN_MS = 1209600000;
```

Comment:
nit: It will be more readable if we assign 14 days value to this const as 60 x60x 24 x 14 x 1000 similar to here https://github.com/mozilla-services/screenshots/blob/master/server/src/config.js#L202

Reply:
Updated the patch to use the default.  Thanks.

Reply:
LGTM!